scalalib/src/scoverage/ScoverageModule.scala

```diff
@@ -0,0 +1,48 @@
+package mill.lib.scoverage
+
+import mill._
+import mill.scalalib._
+
+import _root_.scoverage.Serializer.{ coverageFile, deserialize }
+import _root_.scoverage.IOUtils.{ findMeasurementFiles, invoked }
+import _root_.scoverage.report.ScoverageHtmlWriter
+
+trait ScoverageModule { outer: ScalaModule =>
+
+  trait ScoverageCompile extends ScalaModule {
+    def dataDir: String
+    def sources = outer.sources
+    def resources = outer.resources
+    def scalaVersion = outer.scalaVersion()
+    def compileIvyDeps = outer.compileIvyDeps()
+    def ivyDeps = outer.ivyDeps() ++
+      Agg(ivy"org.scoverage::scalac-scoverage-runtime:1.3.1")
```

Comment:
Besides the fact, that you want to move this part into the worker, the duplicate appearance of the hard coded version number indicates, that it is better moved into a dedicated target.

Reply:
With "it" I mean, it might be a good idea to add a `def scoverageVersion: T[String]` target and use that in the dependencies.

Reply:
Ah, I see. I think you're right.

Reply:
Regarding the worker, after digging in the source tree here, it seems I can dig into scalanativelib, scalalib, and scalajslib to see how they are resolving their dependencies + injecting them into the classpath